tempesta_fw/http_parser.c

```diff
@@ -297,19 +339,30 @@ __FSM_STATE(st) {							\
 	TFW_PARSER_BLOCK(st);						\
 }
 
+#define __FSM_I_TX_COND_nofixup(st, condition, st_next)			\
+__FSM_STATE(st) {							\
+	if (likely(condition))						\
+		__FSM_I_MOVE_nofixup_n(st_next, 1);			\
```

Comment:
Analogous to `__FSM_MOVE_nofixup()`, perhaps it makes sense to introduce `__FSM_I_MOVE_nofixup()` as well?